Android/ZensoftHRCRM/app/build.gradle

```diff
@@ -15,6 +15,10 @@ android {
       minifyEnabled false
       proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
     }
+
+    debug {
+      buildConfigField "String", "BASE_URL", "\"${base_url}\""
```

Comment:
this should be for both, debug and release buildtype,
Please move it defaultConfig